docs/astra.tree

```diff
@@ -5,7 +5,30 @@
 <instance-profile id="astra"
                  name="Astra"
                  start-page="Astra.md">
-
     <toc-element topic="Astra.md"/>
+    <toc-element topic="Getting-started.md"/>
+    <toc-element topic="Operators.md">
+        <toc-element topic="Config-options.md"/>
+        <toc-element topic="Recommendations.md"/>
+        <toc-element topic="Monitoring.md"/>
+    </toc-element>
+    <toc-element topic="Architecture.md">
+        <toc-element topic="Components.md">
+            <toc-element topic="Preprocessor.md"/>
+            <toc-element topic="Indexer.md"/>
+            <toc-element topic="Cache.md"/>
+            <toc-element topic="Query.md"/>
+            <toc-element topic="Manager.md"/>
+            <toc-element topic="Recovery.md"/>
+        </toc-element>
+        <toc-element topic="Misc.md">
+            <toc-element topic="Initial-RFC.md"/>
+        </toc-element>
+    </toc-element>
+    <toc-element topic="Dependencies.md">
+        <toc-element topic="Kafka.md"/>
+        <toc-element topic="Zookeeper.md"/>
+    </toc-element>
+    <toc-element topic="Troubleshooting.md"/>
     <toc-element topic="Roadmap.md"/>
 </instance-profile>
```

Comment:
can we just add the newlines :)

Reply:
Hah, this is a generated file so I don't believe manual changes would persist. 😢